cargo-shuttle/src/lib.rs

```diff
@@ -854,8 +854,18 @@ impl Shuttle {
         {
             let dir_entry = dir_entry.context("get directory entry")?;
 
-            // It's not possible to add a directory to an archive
             if dir_entry.file_type().context("get file type")?.is_dir() {
+                let secrets_path = dir_entry.path().join("Secrets.toml");
+
+                // Make sure to add any `Secrets.toml` files in the subdirectories.
+                if secrets_path.exists() {
```

Comment:
I'm not sure we need to since we join `Secrets.toml` to it. I think it is technically still a valid directory name, but in the event that anyone names a dir in their project `Secrets.toml` I think it will just be added as an empty dir to the archive.